platform.md

```diff
@@ -307,6 +311,7 @@ Usage:
   [-stack <stack> ] \
   [-tag <tag>...] \
   [-uid <uid>] \
+  [-layout] \ # analize images in OCI layout format
```

Comment:
```suggestion
  [-layout] \ # sets <layout>
```

Reply:
Could we add this after `-layers`, to keep the list sorted alphabetically?

Reply:
Do we need to add `-layout-dir` as an additional input? With env `CNB_LAYOUT_DIR`?